pkgs/siderolabs/theila/registry.yaml

```diff
@@ -0,0 +1,8 @@
+packages:
+  - type: github_release
+    repo_owner: siderolabs
+    repo_name: theila
+    asset: "theila-{{ .OS }}-{{ .Arch}}"
+    description: Theila is the UI for all things Sidero Labs
+    link: https://github.com/siderolabs/theila
```

Comment:
```suggestion
```

https://github.com/aquaproj/aqua-registry/blob/main/CONTRIBUTING.md#style-guide

> link is unneeded if repo_owner and repo_name are set

https://aquaproj.github.io/docs/reference/registry-config#default-values-if-repo_owner-and-repo_name-are-set